common/flatpak-run-private.h

```diff
@@ -30,6 +30,16 @@
 #include "flatpak-utils-private.h"
 #include "flatpak-exports-private.h"
 
+/* flags enum for org.freedesktop.portal.Documents.AddFull */
+typedef enum {
```

Comment:
We only use them in flatpak-run.c so it should be fine to move this definition there.